File: app/src/main/java/com/babestudios/companieshouse/data/model/search/Address.java
package com.babestudios.companieshouse.data.model.search;

import com.google.gson.annotations.SerializedName;

public class Address {

	@SerializedName("region")
	public String region;

	@SerializedName("postal_code")
	public String postalCode;

	@SerializedName("address_line_1")
	public String addressLine1;

	@SerializedName("locality")
	public String locality;

	@SerializedName("premises")
	public String premises;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/search/Address.kt
package com.babestudios.companyinfouk.data.model.search

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Address (

	@SerializedName("region")
	var region: String? = null,

	@SerializedName("postal_code")
	var postalCode: String? = null,

	@SerializedName("address_line_1")
	var addressLine1: String? = null,

	@SerializedName("locality")
	var locality: String? = null,

	@SerializedName("premises")
	var premises: String? = null

):Parcelable